connectors/utils.py

```diff
@@ -399,3 +400,20 @@ async def func_to_execute(*args, **kwargs):
         return func_to_execute
 
     return wrapper
+
+
+def ssl_context(certificate):
+    """Convert string to pem format and create a SSL context
```

Comment:
A before/after example in the doc string would be great to understand what's happening at a first glance.